2022-01-20 21:18:09,583 INFO    MainThread:23476 [wandb_setup.py:_flush():71] setting env: {}
2022-01-20 21:18:09,583 INFO    MainThread:23476 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2022-01-20 21:18:09,583 INFO    MainThread:23476 [wandb_init.py:_log_setup():371] Logging user logs to C:\Users\yunjc\_python_jupyter\bupyeonggu\bp_road_crack_detection\_model_test\Pytorch-UNet-master\wandb\run-20220120_211809-3uc1trsn\logs\debug.log
2022-01-20 21:18:09,583 INFO    MainThread:23476 [wandb_init.py:_log_setup():372] Logging internal logs to C:\Users\yunjc\_python_jupyter\bupyeonggu\bp_road_crack_detection\_model_test\Pytorch-UNet-master\wandb\run-20220120_211809-3uc1trsn\logs\debug-internal.log
2022-01-20 21:18:09,584 INFO    MainThread:23476 [wandb_init.py:init():404] calling init triggers
2022-01-20 21:18:09,584 INFO    MainThread:23476 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2022-01-20 21:18:09,584 INFO    MainThread:23476 [wandb_init.py:init():460] starting backend
2022-01-20 21:18:09,584 INFO    MainThread:23476 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=spawn, using: spawn
2022-01-20 21:18:09,587 INFO    MainThread:23476 [backend.py:ensure_launched():216] starting backend process...
2022-01-20 21:18:09,634 INFO    MainThread:23476 [backend.py:ensure_launched():221] started backend process with pid: 18600
2022-01-20 21:18:09,635 INFO    MainThread:23476 [wandb_init.py:init():469] backend started and connected
2022-01-20 21:18:09,636 INFO    MainThread:23476 [wandb_init.py:init():533] updated telemetry
2022-01-20 21:18:09,660 INFO    MainThread:23476 [wandb_init.py:init():563] communicating current version
2022-01-20 21:18:10,571 INFO    MainThread:23476 [wandb_init.py:init():568] got version response 
2022-01-20 21:18:10,571 INFO    MainThread:23476 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-01-20 21:18:11,008 INFO    MainThread:23476 [wandb_init.py:init():606] starting run threads in backend
2022-01-20 21:18:13,036 INFO    MainThread:23476 [wandb_run.py:_console_start():1810] atexit reg
2022-01-20 21:18:13,036 INFO    MainThread:23476 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-01-20 21:18:13,037 INFO    MainThread:23476 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-01-20 21:18:13,038 INFO    MainThread:23476 [wandb_run.py:_redirect():1745] Redirects installed.
2022-01-20 21:18:13,038 INFO    MainThread:23476 [wandb_init.py:init():633] run started, returning control to user process
2022-01-20 21:18:13,038 INFO    MainThread:23476 [wandb_run.py:_config_callback():956] config_cb None None {'epochs': 5, 'batch_size': 1, 'learning_rate': 1e-05, 'val_percent': 0.1, 'save_checkpoint': True, 'img_scale': 0.4, 'amp': True}
2022-01-20 21:18:21,630 INFO    MainThread:23476 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 1
2022-01-20 21:18:21,631 INFO    MainThread:23476 [wandb_run.py:_restore():1752] restore
2022-01-20 21:18:22,174 INFO    MainThread:23476 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 873
  total_bytes: 873
}

2022-01-20 21:18:22,454 INFO    MainThread:23476 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 873
  total_bytes: 873
}

2022-01-20 21:18:23,159 INFO    MainThread:23476 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 873
  total_bytes: 5906
}

2022-01-20 21:18:23,270 INFO    MainThread:23476 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 873
  total_bytes: 5933
}

2022-01-20 21:18:23,382 INFO    MainThread:23476 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 873
  total_bytes: 5933
}

2022-01-20 21:18:23,491 INFO    MainThread:23476 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5933
  total_bytes: 5933
}

2022-01-20 21:18:23,603 INFO    MainThread:23476 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5933
  total_bytes: 5933
}

2022-01-20 21:18:23,712 INFO    MainThread:23476 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5933
  total_bytes: 5933
}

2022-01-20 21:18:23,822 INFO    MainThread:23476 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5933
  total_bytes: 5933
}

2022-01-20 21:18:23,933 INFO    MainThread:23476 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5933
  total_bytes: 5933
}

2022-01-20 21:18:24,508 INFO    MainThread:23476 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5933
  total_bytes: 5933
}

2022-01-20 21:18:25,021 INFO    MainThread:23476 [wandb_run.py:_wait_for_finish():1912] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 5933
  total_bytes: 5933
}
local_info {
}

2022-01-20 21:18:26,355 INFO    MainThread:23476 [wandb_run.py:_append_files():2180] logging synced files
